src/com/collibra/command/handlers/ByeCommandHandler.java

```diff
@@ -19,7 +17,7 @@ public ByeCommandHandler(String clientName, long startTime) {
     }
 
     @Override
-    public void handleCommand(PrintWriter outData, ParsedMessage parsedMessage) {
+    public void handleCommand(PrintWriter outData, String parsedMessage) {
         long endTime = System.nanoTime();
         long duration = (endTime - startTime) / 1000000;
         sendMessage(outData, String.format("BYE %s, WE SPOKE FOR %d MS", clientName, duration));
```

Comment:
clientName and startTime should not be members. They should be passed to `handleCommand` as parameters

Reply:
OK, but how to achieve that if Handler interface has void ```handleCommand(PrintWriter outData, String parsedMessage)```?? Or we need to create other handler with different method signature..